src/routes/AllTopics/AllTopics.js

```diff
@@ -83,7 +83,7 @@ export default class AllTopics extends Component {
 
     const { totalTopicsCached } = this.state
 
-    return <FullPageModal>
+    return <FullPageModal topicView>
```

Comment:
i wish there was a way to avoid having page specific props like this. could we change the name to represent more what it is changing about the style for this page? in this case could be something like `fullWidth`?

Reply:
Got it